llvm/lib/Target/Parasol/CMakeLists.txt

```diff
@@ -34,6 +34,7 @@ add_llvm_target(ParasolCodeGen
   ParasolFrameLowering.cpp
   ParasolInstrInfo.cpp
   ParasolISelLowering.cpp
+  ParasolMergeBaseOffset.cpp
```

Comment:
Should probably remove this. Was one of many failed attempts a address calculation optimization to indexed load/store operations.

The really annoying part of this is that frame elimination happens very late, so you have to use MachineInstructions to optimize that. I should now know enough to tackle it, but it's not an immediate priority.